types/async/index.d.ts

```diff
@@ -218,14 +218,25 @@ export function auto<R extends Dictionary<any>, E = Error>(tasks: AsyncAutoTasks
 export function autoInject<E = Error>(tasks: any, callback?: AsyncResultCallback<any, E>): void;
 
 export function retry<T, E = Error>(
-    opts: number | {
-        times: number,
-        interval: number | ((retryCount: number) => number),
-        errorFilter?: (error: Error) => boolean
-    },
-    task: (callback: AsyncResultCallback<T, E>, results: any) => void,
-    callback: AsyncResultCallback<any, E>
-    ): void;
+    opts?:
+        | number
+        | {
+              times?: number;
+              interval?: number | ((retryCount: number) => number);
+              errorFilter?: (error: Error) => boolean;
+          },
+): Promise<void>;
```

Comment:
I'm _pretty sure_ this overload needs the `task` parameter, since it's the only "required" parameter according to the docs, no?

Reply:
Done, thanks! It was not covered by existing or added tests, never though about this. Amended